### PR TITLE
Added support for Ubuntu 20.04 and maintained support for Ubuntu 18.04

### PR DESCRIPTION
**Pull request by phoerselmann**
Automatic Installation now supports Ubuntu 20.04 "focal fossa" added

The automatic installation has been extended to support the installation on Ubuntu 20.04 "focal fossa".
Additionally the support for Ubuntu 18.04 "bionic" has been maintained.

### Added

-   OS-Decision for Ubuntu 20.04 "focal fossa"
-   OS configuration for Ubuntu 20.04 "focal fossa"

### Changed

-   OS-Decision now informs that Ubuntu 18.04 "bionic" has been maintained
- OS configuration for Ubuntu 18.04 "bionic" maintained

### CODE REVIEW

**Comment by phoerselmann** [comment]
Hey @bheisig 
as discussed the support for Ubuntu 18.04 LTS was re-attached again and Ubuntu 20.04 LTS has been added. Both parts are tested and work like clockwork.

File: idoit-install [hunk 1]
```diff
@@ -1396,6 +1431,13 @@ function secureMariaDB {
         -e"UPDATE mysql.user SET Password=PASSWORD('${MARIADB_SUPERUSER_PASSWORD}') WHERE User='${MARIADB_SUPERUSER_USERNAME}';" || \
         abort "SQL statement failed"
 
+    log "Set plugin mysql_native_password for $MARIADB_SUPERUSER_USERNAME"
```
Comment (bheisig) [inline idoit-install:1434]: You can combine the SQL query with the previous one.

Does this change solve the problem with Debian 10?

**Review by bheisig — CHANGES_REQUESTED** [review]
Hi @phoerselmann,

Thanks for your contribution! Your changes are looking good. I've added only a few hints :-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,14 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased][]
+## [0.14][] - 2020-09-15
```
[the remark this answers is not in the document]
Comment (phoerselmann) [reply CHANGELOG.md:8]: Done, will be added in next commit

File: idoit-install [hunk 3]
```diff
@@ -352,9 +352,23 @@ function identifyOS {
         PHP_FPM_UNIT="php-fpm"
     elif [[ "$NAME" = "SLES" && "$VERSION_ID" == 12* ]]; then
         abort "Error: SLES 12 is out-dated. It's not supported anymore. Please upgrade."
+
```
[the remark this answers is not in the document]
Comment (phoerselmann) [reply idoit-install:355]: Next commit ;-)

File: idoit-install (hunk 1 above)
Comment (phoerselmann) [reply idoit-install:1434]: Combined now.
will be added in next commit.
Yeah, this will solve the Debian 10 issue.

**Comment by bheisig** [comment]
@phoerselmann Your PR is merged. Thank you very much for you contribution :)

**Pull request merged**